proteuslib/unit_models/tests/test_reverse_osmosis_1D.py

```diff
@@ -461,27 +386,27 @@ def test_conservation(self, RO_frame):
     @pytest.mark.component
     def test_solution(self, RO_frame):
         m = RO_frame
-        x_interface_in = value(1 / m.fs.unit.nfe)
-        assert (pytest.approx(-1.5596e5, rel=1e-3) == value(m.fs.unit.deltaP_stage[0]))
-        assert (pytest.approx(1.219e-2, rel=1e-3) ==
+        x_interface_in = m.fs.unit.feed_side.length_domain[2]
+        assert (pytest.approx(-1.755e5, rel=1e-3) == value(m.fs.unit.deltaP_stage[0]))
+        assert (pytest.approx(7.992e-3, rel=1e-3) ==
                 value(m.fs.unit.flux_mass_phase_comp[0, x_interface_in, 'Liq', 'H2O']))
-        assert (pytest.approx(1.733e-6, rel=1e-3) ==
+        assert (pytest.approx(2.114e-6, rel=1e-3) ==
```

Comment:
Same here

Reply:
The same applies to this and anywhere else that uses `x_interface_in` as the space index--the index of the actual inlet is skipped.